editions/tw5.com/tiddlers/landingpage/tiddlers.json

```diff
@@ -0,0 +1 @@
nd will change how you think and how you organise your ideas.</p>\n          <a href=\"https://joearms.github.io/#Index\" class=\"inline-flex items-center \">\n            <span class=\"flex-grow flex flex-col pl-4 hover:text-blue-500\">\n              <span class=\"title-font font-medium text-gray-900\">Joe Armstrong</span>\n              <span class=\"text-gray-500 text-sm\">Co-inventor of Erlang</span>\n            </span>\n          </a>\n        </div>\n      </div>\n      <div class=\"p-4 md:w-1/2 w-full\">\n        <div class=\"h-full bg-gray-100 p-8 rounded\">\n          {{landing page qoute icon}}\n          <p class=\"leading-relaxed mb-6\"><strong>TiddlyWiki gets a Gearhead rating of 6 out of 5 (it's that good).</strong>\n\nFinding code that works flawlessly after just two or three years is magical enough but after seven years?!</p>\n          <a href=\"https://www.networkworld.com/article/3028098/tiddlywiki-a-free-open-source-wiki-revisited.html\" class=\"inline-flex items-center pt-4\">\n            <span class=\"flex-grow flex flex-col pl-4 pt-8 hover:text-blue-500\">\n              <span class=\"title-font font-medium text-gray-900\">Mark Gibbs</span>\n              <span class=\"text-gray-500 text-sm\">Network World</span>\n            </span>\n          </a>\n        </div>\n      </div>\n    </div>\n  </div>\n\t\t\t\t<div class=\"flex justify-center items-center pb-8\" style=\"font-size:0.7em;text-align:center;margin:3em auto;\">{{Product Hunt Link}}</div>\n</section>","title":"landing page testemonials","tags":"","modified":"20210423114217652","created":"20210415131406020"},{"text":"   <section id=\"theme\" class=\"text-gray-700 body-font\">\n            <div class=\"container flex flex-col px-5 py-16 mx-auto lg:items-center md:flex-row lg:px-28\">\n                <div class=\"flex flex-col items-start text-left lg:flex-grow md:w-1/2 lg:pl-24 md:pl-16\">\n                    <h2 class=\"mb-1 text-xs font-medium tracking-widest text-black title-font text-green-500\">Customize</h2>\n                    <h1 class=\"title-font sm:text-4xl text-3xl mb-4 font-medium text-gray-900 mt-0\">\n                        Add a little color</h1>\n                    <p class=\"mb-8 text-base leading-relaxed text-left text-gray-700\">Use themes and palletes to customize TiddlyWiki to your tastes. Use one from the prebundled palettes, install a community made or create your own.</p>\n                    <div class=\"flex flex-col justify-left lg:flex-row\">\n\n                    </div>\n                </div>\n\t\t\t\t\t\t\t\t                <div class=\"w-full mb-10 lg:w-5/6 lg:max-w-lg md:w-1/2\">\n                    <img class=\"object-cover object-center rounded\" alt=\"example of themes\"\n                        src=\"https://i.imgur.com/klxNpR9.png\">\n                </div>\n            </div>\n        </section>\n                  ","title":"landing page themes","tags":"","modified":"20210423114034868","created":"20210415093249376"}]
```

Comment:
Why not make them separate tiddlers, so It's easier for editions like tiddloid and tidgi to copy this to make standard/simillar landing pages?

Reply:
Hi @linonetwo I've done so in 024d65027c1eb4fead37394f1e693773c7c1ee07